src/inference_profiling/model_logs_visualization.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<p>Why is there two rows with # of requests = 3?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
A duplicate reading has been taken for no. of requests=3

Reply:
I'll also update the latest files to these commits.